extras/multisigs.json

```diff
@@ -9,10 +9,11 @@
     "maxi_ops": "0x166f54F44F271407f24AA1BE415a730035637325",
     "blabs_ops": "0x02f35dA6A02017154367Bc4d47bb6c7D06C7533B",
     "linearPoolController": "0x75a52c0e32397A3FC0c052E2CeB3479802713Cf4",
-    "foundation": "0x3B8910F378034FD6E103Df958863e5c684072693",
+    "foundation_opco": "0x3B8910F378034FD6E103Df958863e5c684072693",
     "beets_treasury": "0x811912c19eEF91b9Dc3cA52fc426590cFB84FC86",
     "grants_treasury": "0xE2c91f3409Ad6d8cE3a2E2eb330790398CB23597",
-    "vote_incentive_recycling": "0x9ff471F9f98F42E5151C7855fD1b5aa906b1AF7e"
+    "vote_incentive_recycling": "0x9ff471F9f98F42E5151C7855fD1b5aa906b1AF7e",
+    "bizdev": "0xC7E84373FC63A17B5B22EBaF86219141B630cD7a"
```

Comment:
https://forum.balancer.fi/t/bip-525-business-development-funding-proposal-q1-2024-and-retroactive-q4-2023-funding/5481